packages/components/src/toolbar/toolbar-container.js

```diff
@@ -1,6 +1,22 @@
-const ToolbarContainer = ( props ) => (
-	<div className={ props.className }>
-		{ props.children }
-	</div>
-);
+/**
+ * External dependencies
+ */
+import { useToolbarState, Toolbar } from 'reakit/Toolbar';
+
+/**
+ * Internal dependencies
+ */
+import ToolbarContext from './toolbar-context';
+
+function ToolbarContainer( { accessibilityLabel, ...props } ) {
+	// https://reakit.io/docs/basic-concepts/#state-hooks
+	const toolbarState = useToolbarState( { loop: true } );
+
+	return (
+		<ToolbarContext.Provider value={ toolbarState }>
+			<Toolbar { ...toolbarState } aria-label={ accessibilityLabel } { ...props } />
```

Comment:
`aria-label={ accessibilityLabel }` - we could build a helper which translates accessibility-related props to their ARIA version.